README.md

```diff
@@ -37,6 +37,12 @@ Before you install and use the plug-in:
 
 - **(Linux only - CentOS, RHEL)** Download and Install the following via yum: `python3 make gcc-c++`
 
+- This plug-in builds a native binary at install time that is required to interact with Db2. In the event that a new major version of Node is installed, it is required to uninstall and reinstall this plug-in to update the native binary. This step will need to be taken if you encounter the following error:
+
```

Comment:
Suggested rewording for this: 

Note: Reinstallation is required when Node is upgraded to a new major version. When installed, the plug-in builds a native binary that is needed to interact with Db2. If a new major version of Node is installed, you must uninstall and reinstall the IBM Db2 Database Plug-in to update the binary. Carry out this installation if you encounter the following error:

Reply:
Thoughts on this? Thinking that a brief description of the error (as a heading) might make it stand out to those looking for answers. We could still place the full error below the solution for context.
---

### "`odbc_bindings.node` was compiled against a different Node.js version..."

During the installation process, the plug-in builds a native binary which is required to interact with Db2. When Node is updated to a new major version, the native binary must be recompiled for this new version of Node. To fix this error, uninstall and reinstall this plug-in. 

**Note:** Windows might require multiple uninstall attempts to successfully rebuild the binaries during reinstallation.